examples/quickstart-lerobot/lerobot_example/task.py

```diff
@@ -0,0 +1,257 @@
+"""lerobot_example: A Flower / Hugging Face LeRobot app."""
+
+from pathlib import Path
+from typing import Any
+from collections import OrderedDict
+import numpy as np
+
+import torch
+from evaluate import load as load_metric
+from torch.optim import AdamW
+from torch.utils.data import DataLoader
+
+from lerobot.common.datasets.lerobot_dataset import LeRobotDataset
+from lerobot.common.policies.diffusion.configuration_diffusion import DiffusionConfig
+from lerobot.common.policies.diffusion.modeling_diffusion import DiffusionPolicy
+
+import gym_pusht  # noqa: F401
+import gymnasium as gym
+import numpy
+import imageio
+import time
+
+from datasets.utils.logging import disable_progress_bar
+from .lerobot_dataset_partitioner import LeRobotDatasetPartitioner
+from .lerobot_federated_dataset import FederatedLeRobotDataset
+
+disable_progress_bar()
+fds = None  # Cache FederatedDataset
+
+
+# Create a directory to store the training checkpoint.
+timestr = time.strftime("%Y%m%d/%H/%M")
+output_directory = Path("outputs/train/lerobot_federated_example") / timestr
+output_directory.mkdir(parents=True, exist_ok=True)
+
+# Number of offline training steps (we'll only do offline training for this example.)
+# Adjust as you prefer. 5000 steps are needed to get something worth evaluating.
+# training_steps = 5000
+
+log_freq = 250
```

Comment:
It's preferable to either define it as part of the run-config (i.e. the settings in the `pyproject.toml`) or hardcode it in the `train()` function.